x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/v2/TransportRollupAction.java

```diff
@@ -181,26 +180,39 @@ private static XContentBuilder getProperties(XContentBuilder builder, RollupActi
         builder.startObject("properties");
 
         RollupActionGroupConfig groupConfig = config.getGroupConfig();
-        String dateField = groupConfig.getDateHistogram().getField();
-        HistogramGroupConfig histogramGroupConfig = groupConfig.getHistogram();
-        List<MetricConfig> metricConfigs = config.getMetricsConfig();
+        RollupActionDateHistogramGroupConfig dateHistogramConfig = groupConfig.getDateHistogram();
+        String dateField = dateHistogramConfig.getField();
+        String dateIntervalType = dateHistogramConfig.getIntervalTypeName();
+        String dateInterval = dateHistogramConfig.getInterval().toString();
+        String tz = dateHistogramConfig.getTimeZone() != null ? dateHistogramConfig.getTimeZone() :
+            RollupActionDateHistogramGroupConfig.DEFAULT_TIMEZONE;
 
-        // TODO: Add the format of the original field
-        builder.startObject(dateField).field("type", DateFieldMapper.CONTENT_TYPE).endObject();
+        builder.startObject(dateField).field("type", DateFieldMapper.CONTENT_TYPE)
```

Comment:
I am not sure this needs to be in an integration test, or a unit, but I think we should assert the expected mapping with the respective metadata in our tests. I may be mistaken, but I do not see this.